apps/docs/publisher_agent.md

```diff
@@ -17,19 +17,18 @@ Here we open a websocket at `ws://localhost:5216/publish` to receive price updat
 }
 ```
 
-Once you've gotten a StorkAuth key from Stork, you can use the [generate_keys.py](../../python/src/generate_keys/generate_keys.py) script to generate your EVM and/or Stork keys and to build the `keys.json` file. Make sure not to check this file into version control or share it in any way. 
+You can use the [generate_keys.py](../../python/src/generate_keys/generate_keys.py) script to generate your EVM and/or Stork keys and to build the `keys.json` file. Make sure not to check this file into version control or share it in any way. 
```

Comment:
previous process was:
1. get an auth key from Stork
2. generate keypairs and configure
3. send your public key to Stork
4. run publisher agent

new process is:
1. generate keypairs and configure
2. send your public key to stork
3. run publisher agent